src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlFileStreamTest/SqlFileStreamTest.cs

```diff
@@ -147,12 +151,11 @@ public static void AppendFilestream()
             {
                 string connString = new SqlConnectionStringBuilder(DataTestUtility.TCPConnectionString)
                 {
-                    InitialCatalog = SetupFileStreamDB(ref DataTestUtility.FileStreamDirectory, DataTestUtility.TCPConnectionString)
+                    InitialCatalog = SetupFileStreamDB(ref DataTestUtility.FileStreamDirectory, DataTestUtility.TCPConnectionString),
```

Comment:
I realise that this isn't something you added. But, that ref keyword on the first parameter is a bit odd. Does the called function actually ever write to it? if not could it be removed for simplicity?

Reply:
This was not being used, I have removed it.